src/Commands/RunScan.ts

```diff
@@ -198,6 +198,12 @@ export class RunScan implements CommandModule {
       // eslint-disable-next-line no-console
       console.log(scanId);
 
+      if (warnings.length) {
+        logger.warn(
+          `Scan has been started with warnings: ${warnings.join(', ')}`
```

Comment:
Please adjust serialization
Cover it with tests in `RunScan.spec.ts` with `spy` on `logger`
<img width="573" alt="Screenshot 2024-05-13 at 1 30 31 PM" src="https://github.com/NeuraLegion/bright-cli/assets/103932492/6da7539b-a77b-4f53-a8b0-6dfb9e05fa1c">

Reply:
done, should check it on environment as well